app/workers/publishing_api_gone_worker.rb

```diff
@@ -5,6 +5,8 @@ def perform(content_id, alternative_path, explanation, locale, allow_draft = fal
         .new.govspeak_to_html(explanation)
     end
 
+    alternative_path.squish! if alternative_path
```

Comment:
Have changed the implementation in c7b0f842c7185ab007b4bd6f5a8c2b68bb80b4ae

I decided to use `rstrip` at the same time instead of the original `squish` just to explicitly remove only the trailing whitespace, rather than the blanket removal and normalisation of whitespace that `squish` does.